dotCMS/src/main/java/com/liferay/util/FileUtil.java

```diff
@@ -199,12 +200,15 @@ public static void copyFile(File source, File destination, boolean hardLinks) th
                     Logger.warn(FileUtil.class, "Can't create hardLink. source: " + source.getAbsolutePath()
                             + ", destination: " + destination.getAbsolutePath());
                 }
-            } catch (IOException e) {
-
+            }  catch (FileAlreadyExistsException e1) {
+                Logger.debug(FileUtil.class,
+                        "Could not created the hard link, will try copy for source: " + source +
+                        ", destination: " + destination + ". Error message: " + e1.getMessage());
+            } catch (Exception e2) {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)